lib/datahub-client/data_platform_catalogue/client/graphql_helpers.py

```diff
@@ -166,6 +177,28 @@ def parse_tags(entity: dict[str, Any]) -> list[TagRef]:
     return tags
 
 
+def get_refresh_period_from_cadet_tags(
+    tags: list[TagRef],
+    refresh_schedules: list[str] = ["daily", "weekly", "monthly"]
+) -> str:
```

Comment:
For Justice Data I'm returning title case for this e.g. "Monthly" etc

Do you want to change your one or shall I change mine?